src/TunePID.cpp

```diff
@@ -137,8 +137,21 @@ int main(int argc, char** argv) {
 
 	double period = 8.0;
 
-	std::this_thread::sleep_for(300ms); // wait for encoder position data to arrive
-	int32_t starting_angle = can::motor::getMotorPosition(serial);
+	time_point<steady_clock> start = steady_clock::now();
+	int timeout = 300; // in milliseconds
```

Comment:
Quick couple of things, since this `int` is constant it should really be `constexpr int timeout = 300ms`. Note the ms at the end so you don't have to specify it in comments.